### ISSUE TITLE
Insufficient resource type signposting on search returns

### ISSUE THREAD

**Issue by adamlodge** [archesproject/arches#2904]
### User Story
As an Arches end-user, it can be difficult to determine what are the resource types of the resource instances returned in a search if there are multiple resource models within a given Arches instance.  I believe the only signposting currently provided are the icons associated with the models.

The attached image is intended to illustrate.

It would be nice if the resource model name associated to each returned resource instance was viewable - especially in cases there is more than one resource type being queried.![Screen Shot 2018-01-10 at 12.15.13 PM.png](https://images.zenhubusercontent.com/56f150f76caf56c9d38e16f8/ada7c90c-06fd-45b1-8f1b-1e1cace71fa4)

**Comment by mradamcox** [comment]
I agree with this. Plus, the icon library may not have good enough options to distinguish resource models, in which case an admin may want to use a generic icon, or the same icon for more than one resource model.
